.github/guidelines/CODING_GUIDELINES.md

```diff
@@ -0,0 +1,77 @@
+# MetaMask Coding Guidelines
+
+### 1. New Code Should be Typescript
```

Comment:
nit: Update casing here
```suggestion
### 1. New Code Should be TypeScript
```

Reply:
Fixed in this [commit](https://github.com/MetaMask/metamask-extension/pull/20651/commits/57532c023692c4f8650785ab90e29c9f5d8881ca).